### PR TITLE
HTBHF-2380 Tidy up code around the sending of this email, although it…

### PR DESCRIPTION
**Pull request by mikeSmith08**
… has yet to be implemented.

A bit of a random PR with changes that I was going to make as a part of implementing this email, so thought it still worthwhile creating a PR for this anyway. This contains:
 - Renaming ClaimEmailHandler to DetermineEntitlementNotificationHandler in line with PaymentCycleNotificationHandler
 - Add method `sendNoChildrenOnFeedClaimNoLongerEligibleEmail` to send the email, but it's not used anywhere yet.
 - Added Claim status verification to the integration test for this story.

### CODE REVIEW

File: api/src/test/java/uk/gov/dhsc/htbhf/claimant/communications/DetermineEntitlementNotificationHandlerTest.java [hunk 1]
```diff
@@ -43,4 +43,21 @@ public void shouldPutClaimNoLongerEligibleMessageOnQueue() {
                         entry("First_name", claim.getClaimant().getFirstName()),
                         entry("Last_name", claim.getClaimant().getLastName()));
     }
+
+    @Test
+    public void shouldPutChildrenOnFeedClaimNoLongerEligibleMessageOnQueue() {
```
Comment (YetAnotherMatt) [inline api/src/test/java/uk/gov/dhsc/htbhf/claimant/communications/DetermineEntitlementNotificationHandlerTest.java]: `shouldPutNoChildrenOnFeedClaimNoLongerEligibleMessageOnQueue`
Missing 'No' after 'shouldPut'

Comment (mikeSmith08) [reply api/src/test/java/uk/gov/dhsc/htbhf/claimant/communications/DetermineEntitlementNotificationHandlerTest.java]: Thanks, corrected.

File: api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/UpcomingBirthdayEmailHandler.java [hunk 2]
```diff
@@ -48,39 +48,30 @@ public UpcomingBirthdayEmailHandler(@Value("${payment-cycle.number-of-calculatio
...
-                .emailPersonalisation(emailPersonalisation)
-                .build();
-    }
-
-    private EmailMessagePayload buildChildTurnsOneNotificationEmailPayload(PaymentCycle paymentCycle,
+    private EmailMessagePayload buildChildTurnsAgeNotificationEmailPayload(PaymentCycle paymentCycle,
                                                                            PaymentCycleVoucherEntitlement entitlementNextMonth,
-                                                                           boolean multipleChildrenTurningOneInMonth) {
+                                                                           boolean multipleChildrenTurningOneInMonth,
+                                                                           EmailType emailType) {
```
Comment (dwybourn) [inline api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/UpcomingBirthdayEmailHandler.java]: Parameter should be multipleChildrenTurningAgeInMonth

Comment (mikeSmith08) [reply api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/UpcomingBirthdayEmailHandler.java]: Thanks, have corrected.

**Pull request merged**